source/link/linker.cpp

```diff
@@ -445,6 +445,24 @@ static spv_result_t MergeModules(const MessageConsumer& consumer,
       linked_module->AddDebug2Inst(
           std::unique_ptr<Instruction>(inst.Clone(linked_context)));
 
+  for (const auto& module : input_modules)
+    for (const auto& inst : module->debugs3())
+      linked_module->AddDebug3Inst(
+          std::unique_ptr<Instruction>(inst.Clone(linked_context)));
+
+  // If the generated module uses SPIR-V 1.1 or higher, add an
```

Comment:
This is a nice touch.  :-)

Reply:
:-) I didn’t have it at first, but then decided I might as well make use of it.